docs/reference/resp2-to-resp3-replies.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "RESP2 to RESP3 replies migration guide"
+linkTitle: "RESP2 to RESP3 migration"
+weight: 4
+description: RESP2 to RESP3 replies reference for clients developers
+aliases:
+    - /topics/resp2-to-resp3-replies/
+---
+
+In the journey from RESP2 to [RESP3](https://redis.io/docs/reference/protocol-spec/), one of the primary motivations was 
+to streamline the developer experience by simplifying response parsing. Over time, it became evident that developers 
+frequently performed specific data transformations on RESP2 replies. Addressing these transformations directly at the 
+protocol level not only reduces the overhead for developers but also ensures a more consistent and efficient data 
```

Comment:
does this means to tell the story of why RESP3 was created?
i think it should mention something about replies standing for themselves, without the need for the client library to know which command generated them.

Reply:
Updated. Thanks!